docs/deployment/package/README.md

```diff
@@ -0,0 +1,152 @@
+# Package Deployment
+
+This document will guide you through installing Tetragon on your Linux host machines.
+
+## Requirement
+
+### BTF
+
+Many common Linux distributions now ship with BTF enabled and do not require
+any extra work. To check if BTF is enabled on your Linux system, the standard
+location is:
+
+```
+$ ls /sys/kernel/btf/
+```
+
+### systemd
+
+Tetragon will be managed as a systemd service.
+
+## Configuration
+
+Default configurations are shipped with Tetragon packages that local administrators
+can override by using their own configurations inside `/etc/tetragon/` directory.
+
+To read more about how configurations are handled please check [Tetragon Configuration doc](../../configuration/README.md).
+
+
+## Linux Binary Tarball
+
+### Stable versions
+
+TODO complete.
+
+### Unstable Development versions
+
+TODO complete.
+
+### Install
+
+1. Download the latest binary tarball
+
+   TODO complete.
+
+2. Install Tetragon
+
+   ```bash
+   tar -xvf tetragon-v0.8.3-x86_64.tar.gz
+   sudo cp -vRf tetragon-v0.8.3-x86_64/* /
+   ```
+
+3. Install Tetragon service
+
+   ```bash
+   sudo systemctl daemon-reload
+   sudo systemctl enable tetragon
+   ```
+
+4. Start tetragon service
+
+   ```bash
+   sudo systemctl start tetragon
+   ```
+
+### Update
+
+To update Tetragon:
+
+1. Download new tarball
+
+   TODO complete.
+
+2. Stop Tetragon service
+
+   ```bash
+   sudo systemctl stop tetragon
+   ```
+
+3. Remove old Tetragon version
+
+   ```bash
+   sudo rm -fr /usr/lib/systemd/system/tetragon.service
+   sudo rm -fr /usr/local/bin/tetragon
+   sudo rm -fr /usr/local/lib/tetragon/
+   ```
+
+4. Install new Tetragon version
+
+   ```bash
+   tar -xvf tetragon-v0.8.4-x86_64.tar.gz
+   sudo cp -vRf tetragon-v0.8.4-x86_64/* /
+   ```
```

Comment:
I was going through the installation process and it works nicely,

just one nit.. do we need to pack the tetragon-* directory ?
could the tarball be based on '/' directly? like for go: 

`$ rm -rf /usr/local/go && tar -C /usr/local -xzf go1.19.5.linux-amd64.tar.gz`